File: app/src/main/java/com/example/testlunu/net/acu/weather/DailyForecast.java
package com.example.testlunu.net.acu.weather;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class DailyForecast {
    public String getDate() {
        return date;
    }

    public Integer getEpochDate() {
        return epochDate;
    }

    public Sun getSun() {
        return sun;
    }

    public Moon getMoon() {
        return moon;
    }

    public Temperature getTemperature() {
        return temperature;
    }

    public RealFeelTemperature getRealFeelTemperature() {
        return realFeelTemperature;
    }

    public RealFeelTemperatureShade getRealFeelTemperatureShade() {
        return realFeelTemperatureShade;
    }

    public Float getHoursOfSun() {
        return hoursOfSun;
    }

    public DegreeDaySummary getDegreeDaySummary() {
        return degreeDaySummary;
    }

    public List<AirAndPollen> getAirAndPollen() {
        return airAndPollen;
    }

    public Day getDay() {
        return day;
    }

    public Night getNight() {
        return night;
    }

    public List<String> getSources() {
        return sources;
    }

    public String getMobileLink() {
        return mobileLink;
    }

    public String getLink() {
        return link;
    }

    @SerializedName("Date")
    @Expose
    public String date;
    @SerializedName("EpochDate")
    @Expose
    public Integer epochDate;
    @SerializedName("Sun")
    @Expose
    public Sun sun;
    @SerializedName("Moon")
    @Expose
    public Moon moon;
    @SerializedName("Temperature")
    @Expose
    public Temperature temperature;
    @SerializedName("RealFeelTemperature")
    @Expose
    public RealFeelTemperature realFeelTemperature;
    @SerializedName("RealFeelTemperatureShade")
    @Expose
    public RealFeelTemperatureShade realFeelTemperatureShade;
    @SerializedName("HoursOfSun")
    @Expose
    public Float hoursOfSun;
    @SerializedName("DegreeDaySummary")
    @Expose
    public DegreeDaySummary degreeDaySummary;
    @SerializedName("AirAndPollen")
    @Expose
    public List<AirAndPollen> airAndPollen = null;
    @SerializedName("Day")
    @Expose
    public Day day;
    @SerializedName("Night")
    @Expose
    public Night night;
    @SerializedName("Sources")
    @Expose
    public List<String> sources = null;
    @SerializedName("MobileLink")
    @Expose
    public String mobileLink;
    @SerializedName("Link")
    @Expose
    public String link;

}

File: app/src/main/java/com/example/testlunu/net/acu/weather/DailyForecast.kt
package com.example.testlunu.net.acu.weather

import com.example.testlunu.net.acu.weather.decompiled.Moon
import com.example.testlunu.net.acu.weather.decompiled.Night
import com.example.testlunu.net.acu.weather.decompiled.RealFeelTemperature
import com.example.testlunu.net.acu.weather.decompiled.RealFeelTemperatureShade
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class DailyForecast (

    @SerializedName("Date")
    @Expose
    var date: String?,
    @SerializedName("EpochDate")
    @Expose
    var epochDate: Int?,
    @SerializedName("Sun")
    @Expose
    var sun: Sun?,
    @SerializedName("Moon")
    @Expose
    var moon: Moon?,
    @SerializedName("Temperature")
    @Expose
    var temperature: Temperature?,
    @SerializedName("RealFeelTemperature")
    @Expose
    var realFeelTemperature: RealFeelTemperature?,
    @SerializedName("RealFeelTemperatureShade")
    @Expose
    var realFeelTemperatureShade: RealFeelTemperatureShade?,
    @SerializedName("HoursOfSun")
    @Expose
    var hoursOfSun: Float?,
    @SerializedName("DegreeDaySummary")
    @Expose
    var degreeDaySummary: DegreeDaySummary?,
    @SerializedName("AirAndPollen")
    @Expose
    var airAndPollen: List<AirAndPollen>?,
    @SerializedName("Day")
    @Expose
    var day: Day?,
    @SerializedName("Night")
    @Expose
    var night: Night?,
    @SerializedName("Sources")
    @Expose
    var sources: List<String>?,
    @SerializedName("MobileLink")
    @Expose
    var mobileLink: String?,
    @SerializedName("Link")
    @Expose
    var link: String?
)